### DISCUSSION TITLE
Mirrors (a.k.a. Custom Nodes I think)

### DISCUSSION THREAD

**Discussion by jtsage** [StjerneIdioten/I3D-Blender-Addon#143] [Ideas & Feature Requests]
So, it appears to be the default answer on "how can I get mirrors to work" is to copy-pasta a mirror from an existing mod / model, and then manually change the material mapping on the "new" mirrors.

I took that a step further, and just pulled the missing line from the working material and dump it into my i3d when I am getting close to done - of course, if testing mirrors, or just doing a small edit, this is an extra step - would be way cool if the exporter let you force the extra line, or just did it for you (presence of mirrorShader.xml?)

That extra line, from the vanilla case quadtrac, is 

```xml
<Reflectionmap lodDistanceScaling="0.5" viewDistanceScaling="0.5" type="planar" scaling="1" refractiveIndex="10" bumpScale="0.1" shapesObjectMask="4294967295" lightsObjectMask="4294967295"/>
```

**Comment by NMC-TBone** [comment]
This was implemented in #161
